kiwi/volume_manager/btrfs.py

```diff
@@ -387,10 +387,10 @@ def _xml_pretty(self, toplevel_element):
 
     def _create_snapper_quota_configuration(self):
         root_path = os.sep.join([self.mountpoint, '@/.snapshots/1/snapshot'])
-        snapper_default_conf = os.path.normpath(os.sep.join(
-            [root_path, Defaults.get_snapper_config_template_file()]
-        ))
-        if os.path.exists(snapper_default_conf):
+        snapper_default_conf = Defaults.get_snapper_config_template_file(
+            root_path
+        )
+        if snapper_default_conf:
```

Comment:
IMO the `os.path.exists` call can stay

Reply:
the idea was that the string is empty when no files could be found... but the logic is broken as you found out